README.md

```diff
@@ -1,9 +1,9 @@
 # jv-oop
 
-- Create class Machine containing methods `public abstract void doWork()` and `public abstract void stopWork()`.  
-- Create classes `Truck`, `Bulldozer`, and `Excavator` that will inherit from `Machine`.
+- Create class core.basesyntax.Machine containing methods `public abstract void doWork()` and `public abstract void stopWork()`.  
```

Comment:
revert all these changes 
if you don't know how, just copy README file from master branch